README.md

```diff
@@ -0,0 +1,51 @@
+# gol
+
+gol is a Go implementation of Linux commands for using in terminal.
+
+## Usage
+
+For the all configuration
+```bash
+gol help
+```
+#### Commands
+##### Echo
+```bash
+gol echo
+```
+
+##### ls
+```bash
+gol ls
+```
+
+##### mkdir
+```bash
+gol mkdir
+```
+
+##### pwd
+```bash
+gol pwd
+```
+
+##### whoami
+```bash
+gol pwd
+```
+
+#### Global Options
+- --version, -v print the version (default: false)
+
+## Contributing
+First of all, you can see the available developments in the issues tab. Enter the issue you want to develop and inquire about its availability. If a feature you want to add is not in the issues, please open an issue and wait for a response.
+
+Please make sure to update tests as appropriate.
+
+## License
+[MIT](https://choosealicense.com/licenses/mit/)
+
+## Contributors
+<a href = "https://github.com/Tanu-N-Prabhu/Python/graphs/contributors">
```

Comment:
Should be like this `<a href = "https://github.com/yakuter/gol/graphs/contributors">`